mr/src/test/java/org/apache/iceberg/mr/TestCatalogs.java

```diff
@@ -126,7 +128,9 @@ public void testCreateDropTableToLocation() throws IOException {
     Assert.assertEquals(properties.getProperty("location"), table.location());
     Assert.assertEquals(SchemaParser.toJson(SCHEMA), SchemaParser.toJson(table.schema()));
     Assert.assertEquals(PartitionSpecParser.toJson(SPEC), PartitionSpecParser.toJson(table.spec()));
-    Assert.assertEquals(Collections.singletonMap("dummy", "test"), table.properties());
+    Map<String, String> expectedProperties = defaultProperties();
+    expectedProperties.put("dummy", "test");
+    Assert.assertEquals(expectedProperties, table.properties());
```

Comment:
I think it's fine to have tests for specific catalogs, but I don't think that setting this property is a general behavior of all catalogs that we want to validate. I'd remove these tests because this is up to the catalog implementation and it is fine if a catalog implementation doesn't apply this default. It can still be a valid implementation.

Reply:
Are you suggesting to modify all tests to ensure `dummy` key is present instead of validating the actual set of properties?

Reply:
Yes. Tests like these should ensure that the property passed in is present in properties, not that we have the exact set of properties.

Reply:
I think the tests should assert that the expected properties are a subset of actual properties and should not expect or assert that there are any specific default properties added. That's the basic contract here.

Reply:
Makes sense. Updated tests.